libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -533,9 +533,9 @@ def update_sol_yaml(self, worker_obj: list, filepath: str, cortx_image: str,
                 assert False, "The requested data disk is more than" \
                               " the data disk available on the system"
             if log_disk_flag:
-                # the '1' is being added to accumulate the final data disk count
+                # the '2' is being added to accumulate the final data disk count
                 # after excluding the log disks
-                data_devices_f = device_list[cvg_count+1:]
+                data_devices_f = device_list[cvg_count+2:]
```

Comment:
@Kulkarni-Kaustubh As per pragam's comment, should multiplication be used here ?